tendrl/commons/jobs/__init__.py

```diff
@@ -24,7 +24,7 @@ def _process_job(self, raw_job, job_key):
             # Generate a request ID for tracking this job
             # further by tendrl-api
             req_id = str(uuid.uuid4())
-            if tendrl_ns.type == "node":
+            if tendrl_ns.type == "node" or tendrl_ns.type == "monitoring":
```

Comment:
I didn't know about this. I just need one type to target all monitoring jobs to the monitoring module

Is there a detailed spec or a mail thread that gives an idea of :
* what each of these fields mean/serve?
* The conventions followed etc....